sql/hive/src/main/scala/org/apache/spark/sql/hive/test/TestHive.scala

```diff
@@ -110,7 +110,8 @@ class TestHiveContext(sc: SparkContext) extends HiveContext(sc) {
     /** Fewer partitions to speed up testing. */
     protected[sql] override lazy val conf: SQLConf = new SQLConf {
       override def numShufflePartitions: Int = getConf(SQLConf.SHUFFLE_PARTITIONS, "5").toInt
-      override def dialect: String = getConf(SQLConf.DIALECT, "hiveql")
+      // TODO as in unit test, conf.clear() probably be called, actually all we need is reset
```

Comment:
This is the overwrite function for the `SQLConf`, not the `SQLSession`; the default value is `sql` for `getConf(SQLConf.DIALECT)`, but in `TestHive`, we need `hiveql`.